src/web/demo.rkt

```diff
@@ -452,7 +397,8 @@
        (when (eof-object? formula)
          (raise-herbie-error "no formula specified"))
        (parse-test formula)
-       (define hash (sha1 (open-input-string formula-str)))
+       (define hash (sha1 (open-input-string 
+        (string-append (symbol->string 'improve) formula-str))))
```

Comment:
Why use `string-append` instead of `~s`?

Reply:
And why does the caller construct the hash, instead of doing so inside `run-xxx`?